crates/biome_html_analyze/src/lint/nursery/no_sync_scripts.rs

```diff
@@ -0,0 +1,79 @@
+use biome_analyze::{
+    Ast, Rule, RuleDiagnostic, RuleSource, context::RuleContext, declare_lint_rule,
+};
+use biome_console::markup;
+use biome_html_syntax::HtmlOpeningElement;
+use biome_rowan::AstNode;
+use biome_rule_options::no_sync_scripts::NoSyncScriptsOptions;
+
+declare_lint_rule! {
+    /// Prevent the usage of synchronous scripts.
+    ///
+    /// A synchronous script can impact your webpage performance, read more on how to [Efficiently load third-party JavaScript](https://web.dev/articles/efficiently-load-third-party-javascript).
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```html,expect_diagnostic
+    /// <script src=""></script>
+    /// ```
+    ///
+    /// ### Valid
+    ///
+    /// ```html
+    /// <script src="" async></script>
+    /// <script src="" defer></script>
+    /// ```
+    ///
+    pub NoSyncScripts {
+        version: "next",
+        name: "noSyncScripts",
+        language: "html",
+        recommended: false,
+        sources: &[RuleSource::EslintNext("no-sync-scripts").same()],
```

Comment:
Is this relevant for the html analyze? Perhaps inspired?

Reply:
I think it makes sense here.